<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>1052</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DistroFlex</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/icons/distroflex-icon.png</normaloff>:/icons/distroflex-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="distroSpecsArea">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Distro:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="distroName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>distroName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kernel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kernelName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>kernelName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DEname">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DEname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DE Settings:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DEsettings">
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DEsettings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU info:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpuModel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Model name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coresLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cores:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="threadsLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Threads:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU clock:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpuClock">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPU clock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sensors">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sensors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>USB devices:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="usb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>USB devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Memory devices:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="memdev">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>memdev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceEditMount">
        <property name="font">
         <font>
          <family>Droid Sans</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="directoryEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mount">
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceEditUMount">
        <property name="font">
         <font>
          <family>Droid Sans</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="umount">
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UMount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mkfsDeviceEdit">
        <property name="font">
         <font>
          <family>Droid Sans</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mkfsEdit">
        <property name="font">
         <font>
          <family>Droid Sans</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mkfs">
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Make filesystem</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Droid Sans</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0" columnstretch="0,0">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>300</number>
      </property>
      <property name="verticalSpacing">
       <number>170</number>
      </property>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Software</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/software-icon.png</normaloff>:/icons/software-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Audio</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/audio-icon.png</normaloff>:/icons/audio-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Other</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/other-icon.png</normaloff>:/icons/other-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Devices</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/kbd-icon.png</normaloff>:/icons/kbd-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">borders: none;</string>
        </property>
        <property name="text">
         <string>Distro Specs</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/tux.png</normaloff>:/icons/tux.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/kernel-icon.png</normaloff>:/icons/kernel-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Video</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/video-icon.png</normaloff>:/icons/video-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>DE Behavior</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/icons/display-icon.png</normaloff>:/icons/display-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Droid Sans</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
